src/sentry/admin.py

```diff
@@ -325,12 +325,12 @@ def response_add(self, request, obj, post_url_continue=None):
 
 
 class AuditLogEntryAdmin(admin.ModelAdmin):
-    list_display = ("event", "organization", "actor", "datetime")
-    list_filter = ("event", "datetime")
-    search_fields = ("actor__email", "organization__name", "organization__slug")
-    raw_id_fields = ("organization", "actor", "target_user")
+    list_display = ("event", "organization_id", "actor", "datetime")
+    list_filter = ("event", "datetime", "organization_id")
+    search_fields = ("actor__email",)
+    raw_id_fields = ("actor", "target_user")
     readonly_fields = (
-        "organization",
+        "organization_id",
```

Comment:
There are other models in this module that also have `user` and `organization` in the `readonly_fields` and `raw_id_fields` lists. Do those also need to be updated?

Reply:
For sure, when their foreign key gets broken, but I wanted to ensure the changesets followed along the keys I break incrementally than make the full change all at once.